ui/src/components/NavigationBar/NavBar.js

```diff
@@ -0,0 +1,233 @@
+              </Box>
+
+              <Stack
+                fontWeight="600"
+                spacing={10}
+                color="black"
+                justify="center"
+                align="center"
+                display={{ base: "none", md: "flex" }}
+                marginLeft="40px"
+                isInline
+              >
+                <Box position="relative">
+                  <Link href="/visit">
+                    <a>VISIT</a>
+                  </Link>
+                </Box>
+                <Box position="relative">
+                  <Link href="/about">
+                    <a>ABOUT</a>
+                  </Link>
+                </Box>
+                <Box position="relative">
+                  <Link href="/connect">
+                    <a>CONNECT</a>
+                  </Link>
+                </Box>
+                <Box position="relative">
+                  <Link href="/event">
+                    <a>EVENTS</a>
+                  </Link>
+                </Box>
+                <Box position="relative">
+                  <Link href="/sermon">
+                    <a>SERMONS</a>
+                  </Link>
+                </Box>
+                <Box position="relative">
+                  <Link href="/give">
+                    <a>GIVE</a>
+                  </Link>
+                </Box>
+              </Stack>
+              <Stack
+                spacing={8}
+                color="black"
+                justify="right"
+                align="right"
+                alignItems="center"
+                isInline
+              >
+                <Box
+                  fontWeight="600"
+                  color="#1A365D"
+                  display={{ base: "none", md: "flex" }}
+                >
+                  {loggedIn ? (
+                    <Menu>
+                      <MenuButton>
+                        <Text fontWeight="600">{welcomeMsg[1]}</Text>
+                      </MenuButton>
+                      <MenuList>
+                        <MenuItem href="/">View Profile</MenuItem>
+                        <MenuItem href="/">Log Out</MenuItem>
+                      </MenuList>
+                    </Menu>
+                  ) : (
+                    <Link href="/">
+                      <Text fontWeight="600">{welcomeMsg[0]}</Text>
+                    </Link>
+                  )}
+                </Box>
+                <Box>
+                  <Button
+                    ref={btnRef}
+                    onClick={onOpen}
+                    style={{ background: "none" }}
+                  >
+                    <Image
+                      h="3vh"
+                      src={process.env.PUBLIC_URL + "/menu.svg"}
+                      alt="Menu Button"
+                    />
+                  </Button>
+                </Box>
+              </Stack>
+            </Flex>
+          </Container>
+        </Flex>
+      </Flex>
+
+      {currDate == "Tue" ? (
```

Comment:
was this an intended implementation? why are you rendering a watch SC button on a Tuesday?

Reply:
was testing this on tuesday lol forgot to change it to Sunday. Now alrd changed  to sunday